internal/pkg/store/secretsmanager/secretsmanager.go

```diff
@@ -61,6 +61,15 @@ func (s *SecretsManager) CreateSecret(secretName, secretString string) (string,
 	return aws.StringValue(resp.ARN), nil
 }
 
+func (s *SecretsManager) DeleteSecret(secretName string) error {
+	_, err := s.secretsManager.DeleteSecret(&secretsmanager.DeleteSecretInput{
+		SecretId: aws.String(secretName),
+		// RecoveryWindowInDays *int64 TODO specify?
+	})
+
+	return err
```

Comment:
Can we wrap this error since it's an external error: `fmt.Errorf("delete secret %s from secrets manager: %w", secretName, err)`?